Some clarifications to calling server methods in SignalR (#12015)
aspnet/AspNetCore.Docs: aspnetcore/signalr/javascript-client.md

> [!NOTE]
> If you're using Azure SignalR Service in *Serverless mode*, you cannot call hub methods from a client. For more information, see the [SignalR Service documentation](/azure/azure-signalr/signalr-concept-serverless-development-config).

> The `invoke` method returns a JavaScript [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). The Promise is resolved with the return value (if any) when the method on the server returns. If the method on the server throws an error, the Promise is rejected with the error message. Use the `.then` and `.catch` APIs on the Promise itself to handle these cases (or `await` syntax).

Reviewer: ```suggestion
The `invoke` method returns a JavaScript [Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise). The `Promise` is resolved with the return value (if any) when the method on the server returns. If the method on the server throws an error, the `Promise` is rejected with the error message. Use the `then` and `catch` methods on the `Promise` itself to handle these cases (or `await` syntax).
```

> The `send` method returns a JavaScript [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). The Promise is resolved when the message has been sent to the server. If there is an error sending the message, the Promise is rejected with the error message. Use the `.then` and `.catch` APIs on the Promise itself to handle these cases (or `await` syntax).

Reviewer: ```suggestion
The `send` method returns a JavaScript `Promise`. The `Promise` is resolved when the message has been sent to the server. If there is an error sending the message, the `Promise` is rejected with the error message. Use the `then` and `catch` methods on the `Promise` itself to handle these cases (or `await` syntax).
```

> [!NOTE]

> > Using `send` does not wait until the server has received the message, so it is not possible to return data or errors from the server.

Reviewer: ```suggestion
> Using `send` doesn't wait until the server has received the message. Consequently, it's not possible to return data or errors from the server.
```
